fastlane/lanes/build.rb

```diff
@@ -442,12 +442,36 @@ def buildkite_ci?
     ENV.fetch('BUILDKITE', false)
   end
 
-  def upload_build_to_testflight(whats_new_path:)
+  def upload_build_to_testflight(whats_new_path:, distribution_groups:)
     upload_to_testflight(
       skip_waiting_for_build_processing: true,
       team_id: get_required_env('FASTLANE_ITC_TEAM_ID'),
       api_key_path: APP_STORE_CONNECT_KEY_PATH,
-      changelog: File.read(whats_new_path)
+      changelog: File.read(whats_new_path),
+      distribute_external: true,
+      # If there is a build waiting for beta review, we want to reject that so the new build can be submitted instead
+      reject_build_waiting_for_review: true,
+      groups: distribution_groups
```

Comment:
Thanks @spencertransier for the conversations about this approach we had in Day One iOS 👍